examples/scenarios/file.md

```diff
@@ -0,0 +1,44 @@
+# File
+
+Once you have Stanza installed and running from the [quickstart guide](./README.md#quick-start), you can follow these steps to configure a file to send logs via Stanza.
+
+## Prerequisites
+
+On the host that will be sending logs to Stanza, make sure you know the location of the file you wish to gather logs from.
+
+## Configuration
+
+This is an example config file that can be used in the Stanza install directory. It uses a simple [file operator](/docs/operators/file_input.md) to send logs to Google Cloud utilizing a credentials file that can be generated following Google's documentation [here](https://cloud.google.com/iam/docs/creating-managing-service-account-keys).
+
+```yaml
+pipeline:
+  # For more details on the file operator, go to: https://github.com/observIQ/stanza/blob/master/docs/operators/file_input.md
+  - type: file_input
+    include:
+      - /sample/file/path.log
+
+  # For more info on Google Cloud output, go to: https://github.com/observIQ/stanza/blob/master/docs/operators/google_cloud_output.md
+  - type: google_cloud_output
+    credentials_file: /tmp/credentials.json
+```
+
+If your log file uses multiline log messages, you can use the `multiline` field to define a pattern for the beginning of each log message, as in the following example.
```

Comment:
IMO, we should include a sample multi line input message so the user can try it themselves.

Reply:
Added an additional line pointing to examples on the file operator doc page.